README.md

```diff
@@ -4,6 +4,14 @@
 
 <br />
 
+| Engine    | Conformance Tests |
+|-----------|------------------|
+| MiniWDL   | [![MiniWDL Tests](https://img.shields.io/endpoint?url=https://raw.githubusercontent.com/openwdl/wdl/fix-test/shields/miniwdl_shields.json)](https://github.com/openwdl/wdl/actions/runs/17838004047) |
```

Comment:
These should be updated to point to `wdl-1.2`

Reply:
The code will automatically update the branch this is run on:

```
sed_pattern="https://raw.githubusercontent.com/openwdl/wdl/.*/shields/miniwdl_shields.json"
          new_url="https://raw.githubusercontent.com/openwdl/wdl/$BRANCH/shields/miniwdl_shields.json"
          run_pattern="https://github.com/$REPO/actions/runs/[0-9]+"
          new_run="https://github.com/$REPO/actions/runs/$RUN_ID"
          if grep -qE "$badge_pattern" README.md; then
            git pull
            sed -E -i "/miniwdl_shields\.json/s|$sed_pattern|$new_url|g" README.md
            sed -E -i "/miniwdl_shields\.json/s|$run_pattern|$new_run|g" README.md
            git add README.md
          fi 
```